jordy1611/Overlook
# Feature/initial code cleanup

### What’s this PR do?
* Adds coloring to page through scss
* Refactors scss files to use more inheritance and mixins
* Changes some p elements in index.html to labels for accessibility purposes
* Website passes 100% accessibility
* Adds background images to images folder
### Where should the reviewer start?
* All scss files
* index.html lines 14 & 16
### How should this be manually tested?
* Load website up and check if all features are properly formatted
* Read through comments in scss files
###Which tickets does this resolve?
* Resolves #38 
* Resolves #39 
* Resolves #40
